media/js/firefox/new2.js

```diff
@@ -0,0 +1,280 @@
+        }
+    }
+
+    //GA Custom Dimension in Pageview
+    window.dataLayer.push({
+        event: 'set-state',
+        state: state
+    });
+
+    // conditions in which scene2 should not be shown, even when the
+    // #download-fx hash is set
+    noScene2 = (
+        site.platform === 'other' ||    // no download available
+        site.platform === 'ios' ||      // unsupported platform
+        site.platform === 'fxos' ||  // no download available
+        site.platform === 'android'  // download goes to Play Store
+    );
+
+    $document.ready(function() {
+        var $thankYou = $('.thankyou');
+        var hashChange = ('onhashchange' in window);
+
+        // if desktop with download available, re-locate dl button links
+        if (!noScene2 && $('.download-button-wrapper:visible').length > 0) {
+            var $downloadButtonLinks = $('.download-button-wrapper .download-other-desktop').detach();
+            $downloadButtonLinks.css('display', 'block').insertBefore('#firefox-screenshot');
+        }
+
+        if (site.platform === 'android') {
+            $('#download-button-android .download-subtitle').html(
+                $('.android.download-button-wrapper').data('upgradeSubtitle'));
+
+            $downloadStarted.removeClass('invisible');
+            $downloadReady.removeClass('invisible');
+
+            // On Android, skip all the scene transitions. We're just linking
+            // to the Play Store.
+            return;
+        }
+
+        function showScene(scene) {
+            if (scene === 2) {
+                $downloadReady.addClass('hidden');
+                $downloadStarted.removeClass('hidden');
+
+                $thankYou.focus();
+            } else {
+                $downloadStarted.addClass('hidden');
+                $downloadReady.removeClass('hidden');
+            }
+
+            $downloadInteraction.data('scene', scene);
+        }
+
+        // Pull download link from the download button and add to the
+        // 'click here' link.
+        // TODO: Remove and generate link in bedrock.
+        $directDownloadLink.attr(
+            'href', $('.download-list li:visible .download-link').attr('href')
+        );
+
+        $downloadInteraction.on('click', '#direct-download-link, .download-link', function(e) {
+            e.preventDefault();
+
+            var url = $(e.currentTarget).attr('href');
+
+            // An iframe can not be used here to trigger the download because
+            // it will be blocked by Chrome if the download link redirects
+            // to a HTTP URI and we are on HTTPS.
+            function trackAndRedirect(url, virtualUrl) {
+                // Delay to initiate download is required to allow animation
+                // to finish loading in IE. If delay is removed, the DOM will
+                // unload before the animation completes and the page will
+                // stop in a half-animated state.
+                window.setTimeout(function() {
```

Comment:
I'm going to remove it and put the branch on a demo to test in IE. Hopefully you're right. :+1: